app/experimenter/experiments/tests/test_serializers.py

```diff
@@ -867,6 +916,97 @@ def test_serializer_outputs_expected_schema_for_multipref(self):
 
         self.assertCountEqual(serializer.data["arguments"], expected_data)
 
+    def test_serializer_outputs_expected_schema_for_addon_rollout(self):
+        self.maxDiff = None
+        experiment = ExperimentFactory.create(
+            addon_release_url="https://www.example.com/addon.xpi",
+            countries=[],
+            firefox_channel=Experiment.CHANNEL_BETA,
+            firefox_max_version="71",
+            firefox_min_version="70",
+            locales=[],
+            name="Experimenter Name",
+            normandy_slug="normandy-slug",
+            platform=Experiment.PLATFORM_WINDOWS,
+            population_percent=30.0,
+            rollout_type=Experiment.TYPE_ADDON,
+            slug="experimenter-slug",
+            type=Experiment.TYPE_ROLLOUT,
+        )
+        serializer = ExperimentRecipeSerializer(experiment)
+        self.assertDictEqual(
```

Comment:
Oh related to what we were saying about comparing dicts, I think we should be using this method everywhere, but I stopped myself from going and changing every test to use it.  Maybe we can file that as a follow up.